Extensions/quick_tags.js

```diff
@@ -354,8 +372,9 @@ XKit.extensions.quick_tags = new Object({
 
 		var user_tag_array = XKit.extensions.quick_tags.load_tag_prefs();
 		var m_user_tags = XKit.extensions.quick_tags.render_tags_from_array(user_tag_array, true);
+		var m_add_button = XKit.extensions.quick_tags.render_add_bundle_button(true);
```

Comment:
this is the only place you ever call `render_add_bundle_button`? Doesn't it need to be called everywhere else `render_tags_from_array` is called?